src/comp/ProjectTag.tsx

```diff
@@ -19,7 +26,11 @@ class ProjectTag extends React.Component<ProjectTagProp, ProjectTagState> {
   }
 
   render() {
-    return <p className={styles.ptag_wrapper}>{this.state.name}</p>;
+    return (
+      <p className={styles.ptag_wrapper} onClick={this.props.onClick}>
+        {this.state.name}
+      </p>
+    );
```

Comment:
```suggestion
    const name = `タグ: ${this.props.tag_id}`;
    return (
      <p className={styles.ptag_wrapper} onClick={this.props.onClick}>
        {name}
      </p>
    );
```
これ `setState` がどこにもないので状態を使う意味がないっすね

Reply:
APIを叩いていたのがわかりにくかったので、`ComponentDidMount` にTODOを追加しておきました :bow: